vmm/src/cpu.rs

```diff
@@ -861,6 +863,7 @@ impl CpuManager {
             self.cpuid.clone(),
             self.config.kvm_hyperv,
             topology,
```

Comment:
It would be better to fill in a default CPU topology - there is already some code above that forces the topology on mshv. It would be place where the default topology can be set.